darts/models/regression_ensemble_model.py

```diff
@@ -58,25 +59,51 @@ def __init__(self,
         self.regression_model = regression_model
         self.train_n_points = regression_train_n_points
 
-    def fit(self, series: TimeSeries) -> None:
-        super().fit(series)
+    def _split_multi_ts_sequence(self, n: int, ts_sequence: Sequence[TimeSeries]
+                                 ) -> Tuple[Sequence[TimeSeries], Sequence[TimeSeries]]:
+        left = [ts[:-n] for ts in ts_sequence]
+        right = [ts[-n:] for ts in ts_sequence]
+        return left, right
+
+    def fit(self,
+            series: Union[TimeSeries, Sequence[TimeSeries]],
+            past_covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
+            future_covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
+            num_samples: int = 1
```

Comment:
This should be removed

Reply:
`num_samples` only?

Reply:
It is somewhat for ensemble, because you're using `predict()` in the `fit()`, but I get your point. Removing.